app/src/main/java/com/epfl/drawyourpath/challenge/TournamentViewAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package com.epfl.drawyourpath.challenge
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.epfl.drawyourpath.R
+
+
+/**
+ * used in a recycler view to display the [Tournament]
+ */
+class TournamentViewAdapter(private val tournaments: List<Tournament>) :
+    RecyclerView.Adapter<TournamentViewAdapter.ViewHolder>() {
+
+    /**
+     * Custom view holder using a custom layout for tournaments
+     */
+    class ViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+        val name: TextView
+        val description: TextView
+        val date: TextView
+

```

Comment:
Maybe add a list for the tournament member

Reply:
It should be updated as soon as we have participants to display.